app/src/main/res/layout/tools_container_fragment.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
let's name this layout `dashboard_tools_fragment.xml`

Reply:
reason being, this is being positioned as a sub-component of the dashboard. so I'd like the layout names to reflect that hierarchy.